sphinxcontrib/asciinema/__init__.py

```diff
@@ -1,12 +1,123 @@
-__author__ = "Sergei S."
 __copyright__ = "Copyright (C) 2019"
 __license__ = "MIT"
-__version__ = "0.1.5"
+__version__ = "0.2.0"
+
+import hashlib
+import os
+import posixpath
+
+from docutils import nodes
```

Comment:
I guess moving these imports from `setup(app)` to the top level has caused #2. One way to resolve this would be to move them back, and also move all the new class definitions into that function. Perhaps not very nice.